Kernel/FileSystem/ISO9660FS/Inode.cpp

```diff
@@ -194,6 +194,8 @@ UnixDateTime ISO9660Inode::parse_numerical_date_time(ISO::NumericalDateAndTime c
     i32 year_offset = date.years_since_1900 - 70;
 
     // FIXME: This ignores timezone information in date.
+    // FIXME: This calls `VERIFY(month >= 1 && month <= 12)` (but irritatingly allows days 0 and 255),
+    // which means we can crash the kernel by inserting a CD with an invalid recording month.
```

Comment:
Interesting, since I only moved the from_parts parsing code from one class to another.

Reply:
I'm not accusing you of introducing any bugs, sorry :D

This `VERIFY` seems to have existed for a very long time, at least since 81079ae61636c1f7f32f64466d55d82ae72d67d3. Now that this code is being called in the Kernel on unsanitized data, it *definitely* must not crash. I'm not sure how long this has already been the case, it's just that while reading your PR I noticed this oddity.